src/react/components/_dashboard/UpNext/UpNextItem.js

```diff
@@ -39,18 +46,23 @@ const UpNextItem = ({ dispatch, item, index, applicationLayer }) => {
     ga.trackExternalLink('Up Next', 'bCal', item.htmlLink);
   };
 
+  const showItem = () => {
+    return index == expandedItemIndex;
+  };
+
```

Comment:
There's no reason for this to be a function rather than a constant value, as it's the result of props

Reply:
fixed